.github/workflows/lint_test_charts.yaml

```diff
@@ -48,13 +49,35 @@ jobs:
           done
 
       - name: Check for changed installable charts
-        id: list-changed
+        id: changed
         run: |
-          changed=$(ct list-changed --config .github/ct-install.yaml)
-          if [[ -n "$changed" ]]; then
-            echo "::set-output name=changed::true"
+          set -x
+          charts=$(ct list-changed --config .github/ct-install.yaml)
+
+          if [[ -n "$charts" ]]; then
+            echo "::set-output name=charts::true"
+          fi
+          if [[ -n "$(echo -n "$charts" | grep charts/nri-bundle)" ]]; then
+            echo "::set-output name=bundle-changed::true"
           fi
 
+  install-test:
+    name: Test chart installability
+    runs-on: ubuntu-latest
+    if: ${{ needs.lint-test.outputs.charts-changed == 'true' }}
```

Comment:
what about just adding a temporal commit modifing the logging chart (which has outdated versions of library) to check the pipeline fail, then another commit updating the library version of on the chart checking that it passes ?

Reply:
oh but this runs just on the bundle 🤔  not sure if my proposition makes sense

Reply:
No, this runs if any chart is modified.
```yaml
if: ${{ needs.lint-test.outputs.charts-changed == 'true' }}
[...]     
if: ${{ needs.lint-test.outputs.bundle-changed == 'true' }}
```
I cannot test how GitHub actions or renovate bot is going toe behave. Renovate should open a single PR and in that PR we should be able to test it properly.